curvefs/src/metaserver/metastore.cpp

```diff
@@ -866,5 +869,27 @@ bool MetaStoreImpl::InitStorage() {
     return kvStorage_->Open();
 }
 
+void MetaStoreImpl::LoadDeletedInodes() {
```

Comment:
`LoadDeletedInodes` is the responsibility of InodeManager, Each partition has one InodeManager and InodeStorage, you can load deleted inodes with InodeManager and add to trash_ in InodeManager by pass and the deleted inode loaded are all belong to this partition and trash.

Reply:
fix